examples/airflow/docker-compose.yml

```diff
@@ -87,6 +87,7 @@ services:
     environment:
       - MARQUEZ_HOST=marquez
       - MARQUEZ_PORT=5000
+      - REACT_APP_ADVANCED_SEARCH=false
```

Comment:
Do we need a flag for `psql` vs `engine` based search?

Reply:
This is the flag, but I can rename it if it is confusing.